MSAL/src/public/MSALDefinitions.h

```diff
@@ -83,7 +83,12 @@ typedef NS_ENUM(NSInteger, MSALWebviewType)
     MSALWebviewTypeSafariViewController,
     
 #endif
-    /** Use WKWebView */
+    /** Use WKWebView
+     It is recommended to use the default webview configuration setting provided by a public MSAL API.
+     ex:
+     WKWebViewConfiguration *defaultWKWebConfig = [MSALWebviewParameters defaultWKWebviewConfiguration];
```

Comment:
Can we add this for @property (nonatomic, nullable) WKWebView *customWebview; instead?

Reply:
Could you clarify? it is already declared as a class property as per above discussion @Property (class, nonatomic, nullable) WKWebView *customWebview, and here I am just highlighting a usage case where the developer might invoke the property and assign it to a config variable that they would later use to initialize their custom webview?